### PR TITLE
Corrects codestyle

### PR DESCRIPTION
**Pull request by MarcinOrlowski**
Corrects codestyle, some `final var` + a few refactorings to use switch expr.

BLOCKED_BY #940

### CODE REVIEW

**Comment by BFH-ktt1** [comment]
@MarcinOrlowski : could you resolve the conflict?

**Comment by MarcinOrlowski** [comment]
Done.

File: src/main/java/com/cburch/logisim/soc/nios2/Nios2State.java [hunk 1]
```diff
@@ -180,48 +175,56 @@ public int getIenable() {
...
+          estatus = value;
+          break;
+        case 2:
+          bstatus = value;
+          break;
+        case 3:
+          ienable = value;
+          break;
+        default:
+          // FIXME: what to do by default? throw Exception?
```
Comment (BFH-ktt1) [inline src/main/java/com/cburch/logisim/soc/nios2/Nios2State.java:221]: Yes, I think this makes sense, although it never should happen

**Review by BFH-ktt1 — APPROVED** [review]

File: src/main/java/com/cburch/logisim/soc/nios2/Nios2State.java (hunk 1 above)
Comment (MarcinOrlowski) [reply src/main/java/com/cburch/logisim/soc/nios2/Nios2State.java]: Added.

**Review by BFH-ktt1 — APPROVED** [review]

**Pull request merged**